test/controlplane/pod/hostport/hostport.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package hostport
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	operatorOption "github.com/cilium/cilium/operator/option"
+	agentOption "github.com/cilium/cilium/pkg/option"
+	"github.com/cilium/cilium/test/controlplane"
+	"github.com/cilium/cilium/test/controlplane/services/helpers"
+	"github.com/cilium/cilium/test/controlplane/suite"
+)
+
+func init() {
+	suite.AddTestCase("Pod/HostPort", testHostPort)
+
+}
+
+func testHostPort(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	abs := func(f string) string { return path.Join(cwd, "pod", "hostport", f) }
+
+	modConfig := func(daemonCfg *agentOption.DaemonConfig, _ *operatorOption.OperatorConfig) {}
+
+	k8sVersions := controlplane.K8sVersions()
+	// We only need to test the last k8s version
+	test := suite.NewControlPlaneTest(t, "pod-hostport-control-plane", k8sVersions[len(k8sVersions)-1])
```

Comment:
```suggestion
	test := suite.NewControlPlaneTest(t, "hostport-control-plane", k8sVersions[len(k8sVersions)-1])
```

this has to match teh name of the control plane node on the init.yaml file, otherwise it does not start waiting for the local node

Reply:
done!